### PR TITLE
run-kvm-unit-test.sh [Version 2]: enable options for configuring kvm-unit-tests

### PR DESCRIPTION
**Pull request by balamuruhans**
support configuring kvm-unit-test with supported arguments for enabling tests to
run in PowerPC.

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

File: contrib/testsuites/run-kvm-unit-test.sh [hunk 1]
```diff
@@ -21,9 +21,9 @@
 WILDCARD="*"
 while [ true ]; do
     case $1 in
-        "--endian")
+        --configure-args*)
```
Comment (ldoktor) [inline contrib/testsuites/run-kvm-unit-test.sh]: Do you have a special reason for allowing extra characters after the `--configure-args`? I don't think `--configure-args-foo` should be accepted (basically I don't think the tailing `*` should be there, but if you have a special reason for it, please let me know).

File: contrib/testsuites/run-kvm-unit-test.sh [hunk 2]
```diff
@@ -21,9 +21,9 @@
 WILDCARD="*"
 while [ true ]; do
     case $1 in
-        "--endian")
```
Comment (ldoktor) [inline contrib/testsuites/run-kvm-unit-test.sh]: I'd prefer keeping the `--endian` supported as some people might be using it...

File: contrib/testsuites/run-kvm-unit-test.sh [hunk 3]
```diff
@@ -34,12 +34,12 @@ while [ true ]; do
             WILDCARD="$1"
             ;;
         "-h"|"--help")
-            echo "Usage: $0 [-h] [--endian ENDIAN] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
+            echo "Usage: $0 [-h] [--configure-args ['--endian ENDIAN][kvm-unit-test configure arguments]...'] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]"
```
Comment (ldoktor) [inline contrib/testsuites/run-kvm-unit-test.sh]: This line is corrupted. It should be:

```
Usage: $0 [-h] [--configure-args KVM_UNIT_TEST_CONFIGURE_ARGS] [--path PATH] [--wildcard WILDCARD] [avocado arguments ...]
```

or simply `[--configure-args CONFIGURE_ARGS]`, people can get the details from the help below...

**Comment by balamuruhans** [comment]
@ldoktor have made changes as per your suggestion.

**Pull request merged**